pkgs/tools/text/bashblog/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, lib, fetchzip, fetchFromGitHub, makeWrapper, bash, pandoc }:
+
+stdenv.mkDerivation rec {
+  pname = "bashblog";
+  version = "unstable-2022-03-26";
+
+  src = fetchFromGitHub {
+    owner = "cfenollosa";
+    repo = "bashblog";
```

Comment:
This should be `pname` instead of `"bashblog"`

Reply:
Although this is true, I prefer to leave this static.

Reply:
Why? Using `pname` is a standard across the repository where the pname and repo name match.

Reply:
To be honest, I haven't seen this as a standard, or at least I haven't got issues with using the static method on PRs that I have submitted. But the main reason I use it is mostly in case there are changes in the repository name.

Reply:
Alright, I guess you can leave it then. LGTM!

Reply:
No it's not. People startet to do this but replacing the name of the program everywhere with pname just makes things harder.